Jenkinsfile

```diff
@@ -0,0 +1,259 @@
+                        }
+                    }
+                }
+        }
+    }
+
+    if(config.dockerScan){
+        stage('Twistloc scan') {
+            withDockerServer([uri: dockerHost()]) {
+                config.dockerRepos.each { dockerRepo ->
+                                            dockerName = config.dockerRegistry+dockerRepo+":${config.docker_tag}"
+                                            echo "Twistloc Scan ${dockerName}"
+                                            twistlockScan   ca: '',
+                                                            cert: '',
+                                                            compliancePolicy: 'critical',
+                                                            dockerAddress: dockerHost(),
+                                                            gracePeriodDays: 0,
+                                                            ignoreImageBuildTime: true,
+                                                            image: dockerName,
+                                                            key: '',
+                                                            logLevel: 'true',
+                                                            policy: 'warn',
+                                                            requirePackageUpdate: false,
+                                                            timeout: 10
+                }
+            }
+        }
+    }
+
+    if(config.dockerScan){
+        stage('Twistloc publish') {
+            withDockerServer([uri: dockerHost()]) {
+                config.dockerRepos.each { dockerRepo ->
+                                            dockerName = config.dockerRegistry+dockerRepo+":${config.docker_tag}"
+                                            echo "Twistloc Publish ${dockerName}"
+                                            twistlockPublish    ca: '',
+                                                                cert: '',
+                                                                dockerAddress: dockerHost(),
+                                                                ignoreImageBuildTime: true,
+                                                                image: dockerName,
+                                                                key: '',
+                                                                logLevel: 'true',
+                                                                timeout: 10
+                }
+            }
+        }
+    }
+
+    if (!config.isPrJob) {
+        stage('Publish Docker Images') {
+            withDockerServer([uri: dockerHost()]) {
+                config.dockerRepos.each { dockerRepo ->
+                    sh "docker tag ${config.dockerRegistry}${dockerRepo}:${config.docker_tag} ${config.dockerRegistry}${dockerRepo}:master-latest"
+                    sh "docker push ${config.dockerRegistry}${dockerRepo}:${config.docker_tag}"
```

Comment:
just wondering, why do we publish this one if it's not a release?

Reply:
for each build we usually publish to ECR so you can get the latest image version that is in development.